src/attr/_make.py

```diff
@@ -1068,6 +1068,32 @@ def fields(cls):
     return attrs
 
 
+def fields_dict(cls):
+    """
+    Returns an OrderedDict of ``attrs`` attributes for a class, whose keys are
+    the attribute names.
+
+    :param type cls: Class to introspect.
+
+    :raise TypeError: If *cls* is not a class.
+    :raise attr.exceptions.NotAnAttrsClassError: If *cls* is not an ``attrs``
+        class.
+
+    :rtype: OrderedDict where keys are attribute names and values are
```

Comment:
In this case it’s worth explaining what happens (something along of “a plain dict if it's naturally ordered like in Python 3.6+, `OrderedDict` otherwise”)